### PR TITLE
Feature/Foundation extensions

### PR DESCRIPTION
**Pull request by borut-t**
Some more Foundation extensions.

### CODE REVIEW

File: Sources/Extensions/Foundation/Double+Povio.swift [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  /// Returns miles in meters
+  var meters: Meters {
+    self * 1609.344
+  }
+  
+  /// Return meters in miles
+  var miles: Miles {
+    self / 1609.344
+  }
+}
```
Comment (nejcvivod) [inline Sources/Extensions/Foundation/Double+Povio.swift:24]: Not sure I like this, we can easily get into a situation where we do the same conversion twice :\ (meters to meters for example)

Should we have proper types wrapping this and then make conversions between those?

File: Sources/Extensions/Foundation/Dictionary+Povio.swift [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+public extension Dictionary {
+  /// Make a union on two dictionaries.
+  /// ```
+  /// let dictionaryA = [1: "One", 2: "Two"]
+  /// let dictionaryB = [3: "Three"]
+  /// let unionDictionary = dictionaryA.union(dictionaryB)
+  /// print(unionDictionary) // [1: "One", 2: "Two", 3: "Three"]
+  /// ```
+  func union(_ dictionary: Dictionary) -> Dictionary {
```
Comment (tonikocjan) [inline Sources/Extensions/Foundation/Dictionary+Povio.swift:19]: What happens if dictionaries contain the same key. Which value remains?

File: Sources/Extensions/Foundation/String+Povio.swift [hunk 3]
```diff
@@ -37,4 +37,14 @@ public extension String {
...
+  
+  /// Returns first n characters from string
+  func first(n characters: Int) -> String {
+    .init(prefix(min(count, characters)))
+  }
+  
+  /// Returns last n characters from string
+  func last(n characters: Int) -> String {
+    .init(suffix(min(count, characters)))
+  }
```
Comment (tonikocjan) [inline Sources/Extensions/Foundation/String+Povio.swift:49]: These seem redundant, `prefix` and `suffix` already do their job, do they not?

File: Sources/Extensions/Foundation/Dictionary+Povio.swift (hunk 2 above)
Comment (borut-t) [reply Sources/Extensions/Foundation/Dictionary+Povio.swift:19]: Original is preserved. Which is not good. I'll think about it.

File: Sources/Extensions/Foundation/Double+Povio.swift (hunk 1 above)
Comment (borut-t) [reply Sources/Extensions/Foundation/Double+Povio.swift:24]: Thinking further about this if this makes sense at all. I'll think about it.

File: Sources/Extensions/Foundation/String+Povio.swift (hunk 3 above)
Comment (borut-t) [reply Sources/Extensions/Foundation/String+Povio.swift:49]: Yes, this is from pre-swift4 times.

File: Sources/Extensions/Foundation/Dictionary+Povio.swift (hunk 2 above)
Comment (borut-t) [reply Sources/Extensions/Foundation/Dictionary+Povio.swift:19]: Actually, this is redundant. [Dictionary](https://developer.apple.com/documentation/swift/dictionary/3127169-merge) already has `merge` and `merging` methods.

File: Sources/Extensions/Foundation/Double+Povio.swift (hunk 1 above)
Comment (borut-t) [reply Sources/Extensions/Foundation/Double+Povio.swift]: @nejcvivod I've decided to remove it since it's not clear what it does with the current state.

**Pull request merged**